src/ckb/ckb-info.plist

```diff
@@ -5,9 +5,11 @@
     <key>CFBundleExecutable</key>
     <string>ckb</string>
     <key>CFBundleIconFile</key>
-    <string>ckb-logo.icns</string>
+    <string>@ICON@</string>
     <key>CFBundleName</key>
     <string>ckb</string>
+    <key>CFBundleShortVersionString</key>
+    <string>@SHORT_VERSION@</string>
```

Comment:
Same for this piece of code.
It is done like that on `cmake` branch, but it has instructions in the listfile to substitute for the proper name.

Reply:
Likewise as above, this is all explained in Qt documentation.